src/map/mob_modifier.h

```diff
@@ -98,6 +98,7 @@ enum MOBMODIFIER : int
     MOBMOD_ALLI_HATE          = 68, // Range around target to add alliance member to enmity list.
     MOBMOD_NO_LINK            = 69, // If set, mob cannot link until unset.
     MOBMOD_NO_REST            = 70, // Mob cannot regain hp (e.g. re-burrowing antlions during ENM).
+    MOBMOD_SEVERE_CHANCE      = 71, // % chance to use a severe spell like death or impact
```

Comment:
don't forget this one

Reply:
Hmmm. Slot 13 is occupied by MOBMOD_RAGE // define rage timer -- only add as a spawn mod! (OBSOLETE - use rage mixin)
Even though it's obsolete, is it safe to overwrite this?

Reply:
then 13 should have been mobmod rage in status.lua also.

@ibm2431 @zach2good more desync between core and lua..

Reply:
looks liek ALL the ones marked "available for use are in fact NOT :rage:

Reply:
As discussed I'll leave this at 71 for now. The obsolete mods should be discussed independently.